### ISSUE TITLE
Add Similar Skills, Skills By Author Section

### ISSUE THREAD

**Issue by fragm3** [fossasia/susi_skill_cms#758]
**Expected Behaviour**

More By This Author:

<img width="1064" alt="screen shot 2018-06-16 at 11 00 57 pm" src="https://user-images.githubusercontent.com/18073126/41500974-37bb2e56-71b9-11e8-8921-3c51221a94f8.png">

Similar Skills can be made similar to this(without rating and prices):

<img width="1313" alt="screen shot 2018-06-16 at 11 01 51 pm" src="https://user-images.githubusercontent.com/18073126/41500978-4c07bece-71b9-11e8-8530-1901188066e1.png">

API needs to be created for similar skills

**Would you like to work on the issue?**

YES

**Comment by AnupKumarPanwar** [comment]
We have skills by author already implemented. You can click on the author's name on the skill page to see the list of skills create by that author.

**Comment by fragm3** [comment]
Yes, I know that. This feature should be displayed on page itself, its better this way

**Comment by anshumanv** [comment]
@fragm3 progress?

**Comment by fragm3** [comment]
@anshumanv the API is not implemented yet for this
